### PR TITLE
Feat/shiden crowdloan

### PR DESCRIPTION
**Pull request by sirius651**
implement logic :: including #73 

currently, it's for Osaka endpoint, for publishing to main, we need change as Kusama one.

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -11,6 +12,11 @@ const routes: RouteRecordRaw[] = [
     name: 'Main',
     component: Main
   },
+  {
+    path: '/shiden',
```
Comment (hoonsubin) [inline src/router/index.ts:16]: I recommend that we auto direct the active crowdloan page when the user enters from the root dir.
So the Polkadot crowdloan should be `/astar` and Kusama be `/shiden` and we switch the current active crowdloan to be redirected from `/`

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/router/index.ts (hunk 1 above)
Comment (sirius651) [reply src/router/index.ts:20]: Fixed

**Review by hoonsubin — APPROVED** [review]
LGTM

**Review by bobo-k2 — APPROVED** [review]
I checked everything, but staking because I don't have KSM. 
LGTM

**Review by impelcrypto — APPROVED** [review]
LGTM. Please review [PR#76](https://github.com/AstarNetwork/crowdloan-app/pull/76) before merging this PR if possible.

**Pull request merged**